tests/system_test.sh

```diff
@@ -6,25 +6,48 @@
 #
 # Needs to be run from the root directory of the project.
 
+
+
+echo "travis_fold:start:docker-build"
+echo -e "\e[33;1mBuild docker images\e[0m"
 docker-compose build || { printf ">>> ERROR: Docker build failed"; exit 2; }
+echo "travis_fold:end:docker-build"
+
+echo "travis_fold:start:docker-up"
+echo -e "\e[33;1mStarting containers\e[0m"
 docker-compose up -d || { printf ">>> ERROR: Launching containers failed"; exit 2; }
+echo # `echo` to work around formatting issues after `docker-compose`
+echo "travis_fold:end:docker-up"
 
 # This is guestimate for the time when the system goes up and operational.
 # Doing this other ways is much more complex. May need to be increased on slow boxes.
 sleep 5
```

Comment:
Is it possible to get rid of that sleep?

Reply:
Indeed, I couldn't do it quickly before. Will look into it today

Reply:
I could appreciate if could throw it out. Probably, you could send a simple request repeatedly to eval if the system is up, but this adds a lot of complexity. Other ideas?